.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: Build
+
+on:
+  pull_request:
+    branches: ['**']
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        consul:
+          - 1.1.1
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build and Push Docker Image
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
+        with:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+          BASE_TAG: ${{ matrix.consul }}
```

If we don't plan on building for more than one consul version, we could also just pass `1.1.1` here.
```suggestion
          BASE_TAG: 1.1.1
```

---

(and get rid of the matrix stuff on L12-14)

---

Yeah, good catch. I was thinking that as we upgrade Consul internally, we'll probably want to add versions to the matrix, so I left it in.